unstructured/partition/pptx.py

```diff
@@ -417,7 +384,9 @@ def increment_page_number(self) -> Iterator[PageBreak]:
             yield PageBreak(
                 "",
                 detection_origin=DETECTION_ORIGIN,
-                metadata=ElementMetadata(last_modified=self.last_modified),
+                metadata=ElementMetadata(
+                    last_modified=self.last_modified, page_number=self.page_number - 1
```

Comment:
Is this a bug fix (the `page_number` for `PageBreak` elements)?

Reply:
Yes, I'd say so. No one has ever complained about it as far as I know, but the idea is that hash element-ids need to be unique within a single document and this was causing all PageBreak elements to get the same hash-id.

The hash-id is based on text, filename, page-number, and sequence-number within that page. When `PageBreak` did not have `.metadata.page_number`:
- text was always "" as it is for any `PageBreak` element.
- filename was the same for all elements in the file
- page_number was always the same (`None`)
- A missing page-number causes the page-sequence counter to reset, so page_seq_num would always be 1

Consequently the hash for a `PageBreak` would always be sha1(("", filename, None, 1)) which is constant for a given file.